stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -69,6 +70,12 @@ public class MessageListViewModel(
      */
     private var threadListData: MessageListItemLiveData? = null
 
+    /**
+     * Regulates the visibility of deleted messages.
+     */
+    @InternalStreamChatApi
+    public var deletedMessageVisibility: MutableLiveData<DeletedMessageVisibility> = MutableLiveData(DeletedMessageVisibility.ALL)
```

Comment:
Visibility set to public because otherwise it is inaccessible in `MessageListViewModel.bindView()`